gem_metrics/bertscore.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+from .metric import ReferencedMetric
+from datasets import load_metric
+import numpy as np
+
+
+class BERTScore(ReferencedMetric):
+    """BERTScore uses the tiny checkpoint for efficient CPU runtime."""
+    def __init__(self):
+        """Load the BERT checkpoint into memory."""
+        self.metric = load_metric('bertscore')
+
+    def _make_serializable(self, score_entry):
+        """Convert from tensor object to list of floats."""
+        return [float(score) for score in score_entry.numpy()]
+
+    def compute(self, predictions, references):
+        """If it can run, return BLEURT, otherwise skip."""
```

Comment:
How will it skip? This looks like it'll always run?

Reply:
oops, copy paste error :) I made it mandatory to run since we are only using a small model. Taking out the comment.